libglusterfs/src/glusterfs/glusterfs.h

```diff
@@ -315,6 +315,7 @@ enum gf_internal_fop_indicator {
 #define DHT_SKIP_OPEN_FD_UNLINK "dont-unlink-for-open-fd"
 #define DHT_IATT_IN_XDATA_KEY "dht-get-iatt-in-xattr"
 #define DHT_MODE_IN_XDATA_KEY "dht-get-mode-in-xattr"
+#define GF_MKNOD_REPLACE_KEY "mknod-replace-if-linkto"
```

Comment:
Why not just this:

```suggestion
#define GF_FORCE_REPLACE_KEY "force-replace"
```

This way the name is more generic and it could be easily reused in the future if necessary for other purposes not related to linkto files, and for other fops, not only mknod.